recipe-client-addon/lib/NormandyDriver.jsm

```diff
@@ -31,8 +31,11 @@ this.NormandyDriver = function(sandboxManager) {
   }
   const {sandbox} = sandboxManager;
 
+  Storage.seedDurability(sandbox);
```

Comment:
> this will count only the first recipe run the first time as non-durable

I'm not sure why this is would be the case - the durability value is seeded to `1` (if none is previously set) and then the driver checks that value when it creates its Storage. In theory, the first recipe/run should have a `durability` of `1` and should run as expected (assuming the store is durable).

However, In selfrepair, the durability check looks for a value of `2` instead, which didn't really add up to me. Maybe there's something about the recipe life cycle that I'm missing.

 > It would probably be good to call seedDurability from the context of RecipeRunner

 I'm confused how this would work with regards to the sandboxes. Would we need to seed each action/sandbox in `RecipeRunner.executeAction`? Would the call to `Storage.checkDurability` then still exist in NormandyDriver?